ui/lobby/src/view/realTime/chart.ts

```diff
@@ -57,7 +57,14 @@ function renderPlot(ctrl: LobbyController, hook: Hook) {
           mouseOnToPopup: true,
           closeDelay: 200,
           popupId: 'hook'
-        }).data('powertipjq', $(renderHook(ctrl, hook)));
+        }).data('powertipjq', $(renderHook(ctrl, hook)))
+          .on({
+            powerTipRender() {
+              $('.inner-clickable').click(() => {
+                ctrl.clickHook(hook.id);
+              });
```

Comment:
have you tested this? It seems to me that this selects ALL hooks and makes them all accept the same one

Reply:
> have you tested this?
Tested on this popup. Should I test that other popups are still working?

> It seems to me that this selects ALL hooks and makes them all accept the same one
Not sure what you mean, sorry. Unless I misunderstand something, this only runs on 
1. The hook handled by `renderPlot`
2. An element with class `inner-clickable`

The general design is the same as this, as far as I can tell: https://github.com/ornicar/lila/blob/master/ui/site/src/standalones/util.js#L138

Reply:
`.inner-clickable'` selects any and every `.inner-clickable'` element of the page. There can be more.